README.md

```diff
@@ -329,23 +341,24 @@ This is a known issue that is being actively worked on.
 const { DynamicRemotePlugin } = require('@openshift/dynamic-plugin-sdk-webpack');
 
 const sharedModules = {
-    '@openshift/dynamic-plugin-sdk':( singleton: true},
-    '@scalprum/react-core':( singleton: true},
-    react: { singleton: true),
-    'react-dom': ( singleton: true },
+  '@openshift/dynamic-plugin-sdk': { singleton: true, eager: true },
```

Comment:
We don't want `eager` config in module packages. Eager means the package will be always included in the initial chunks which increases the size. The host should always load the critical packages first anyway. If anything we want the `import: false` for packages like react. because we don't really need these packages imported via the remote modules.